dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/osgi/response/IResponseImpl.java

```diff
@@ -45,6 +46,9 @@ public class IResponseImpl implements IResponse
   
   private static final TraceComponent tc = Tr.register(IResponseImpl.class, WebContainerConstants.TR_GROUP, WebContainerConstants.NLS_PROPS);
 
+  private static final HttpHeaderKeys CONTENT_TYPE_KEY = HttpHeaderKeys.HDR_CONTENT_TYPE;
```

Comment:
Again not sure why making static finals on this class instead of referencding HttpHeadersKeys static finals in the code.

Reply:
fixed thanks